src/App.js

```diff
@@ -9,9 +9,10 @@ import Profile from './pages/profile/profile.component'
 import './App.css';
 import CurrentUserContext from './contexts/current-user/current-user.context';
 import ForgotPassword from './components/forgot-password/forgot-password.component';
-
-import {createStructuredSelector} from 'reselect'
+import {getItem} from './utils'
```

Comment:
Follow a single convention.  either add spacing between brackets in all or remove between all.